glyphs2fontir/src/source.rs

```diff
@@ -300,8 +303,21 @@ impl Work<Context, WorkError> for StaticMetadataWork {
             .filter(|gn| self.glyph_names.contains(gn))
             .collect();
 
+        let selection_flags = match font.use_typo_metrics.unwrap_or_default() {
+            true => SelectionFlags::USE_TYPO_METRICS,
+            false => SelectionFlags::empty(),
+        } |
+        // https://github.com/googlefonts/glyphsLib/blob/42bc1db912fd4b66f130fb3bdc63a0c1e774eb38/Lib/glyphsLib/builder/names.py#L27
+        match font.default_master().name.to_ascii_lowercase().as_str() {
```

Comment:
I'm afraid simply looking at the default master.name is not sufficient, because glyphsLib contains additional logic, see `build_stylemap_names` (that is called when exporting glyphs->ufo by fontmake because `minimize_glyphs_diffs` is False when compiling):

https://github.com/googlefonts/glyphsLib/blob/42bc1db912fd4b66f130fb3bdc63a0c1e774eb38/Lib/glyphsLib/builder/names.py#L27-L43

the final ufo.info.styleMapStyleName in the end is what matters to decide the style flags for ufo2ft, and the default UFO master that glyphsLib produces is going to be the default master TTF which varLib will duplicate to use as basis for the new variable font.

Reply:
This part I would prefer to kick down the road slightly, pushed to https://github.com/googlefonts/fontmake-rs/issues/300

Reply:
fair enough